src/__experimental__/components/date-range/date-range.stories.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import notes from './documentation';
+import DateRange from './date-range.component';
+
+const store = new Store({
+  value: ['2016-10-01', '2016-10-30']
+});
+const handleChange = (newDate) => {
+  store.set({ value: newDate });
+  action('changed')(newDate);
+};
+
+storiesOf('Experimental/Date Range', module)
+  .addParameters({
```

Comment:
Do we want to keep that in decorator when we have only one story for the component?

Reply:
you're right, I'll move it down